mirgecom/initializers.py

```diff
@@ -262,6 +263,100 @@ def __call__(self, x_vec, *, t=0, eos=IdealSingleGas()):
                               momentum=mom)
 
 
+class DoubleMachReflection:
+    r"""Implement the double shock reflection problem.
+
+        - Woodward and Collela
+
+    The inital condition is defined
+
+    .. math::
+
+        (\rho,u,v,P) =
+
+    This function only serves as an initial condition
+
+    .. automethod:: __init__
+    .. automethod:: __call__
+    """
+
+    def __init__(
+            self, dim=2, shock_location=1.0/6.0, shock_speed=4.0
+    ):
+        """Initialize initial condition options.
+
+        Parameters
+        ----------
+        dim: int
+           dimension of domain, must be 2
+        shock_location: float
+           location of shock
+        shock_speed: float
+           shock speed, Mach number
+        """
+        self._shock_location = shock_location
+        self._dim = dim
+        self._shock_speed = shock_speed
+
+    def __call__(self, x_vec, *, t=0, eos=IdealSingleGas()):
+        """
+        Create the 1D Sod's shock solution at locations *x_vec*.
```

Comment:
Leftover from copying `SodShock1D`?

Reply:
This (and a few other pieces) are left-over from that case. I will clean it up.

Reply:
Fixed: https://github.com/illinois-ceesd/mirgecom/pull/213/commits/974bcd47752630f73c6b9a536352b4bc128a5b2c